elasticdl/python/common/k8s_job_monitor.py

```diff
@@ -43,24 +44,28 @@ def monitor_status(self):
         pod_succeeded = False
 
         while True:
-            pod = self.client.get_pod(self.pod_name)
-            if pod is None:
-                retry_num += 1
-                if retry_num > MAX_READ_POD_RETRIES:
-                    logger.error("{} Not Found".format(self.pod_name))
+            try:
+                pod = self.client.get_pod(self.pod_name)
```

Comment:
Will `self.client.get_pod(self.pod_name)` throw exception.

Reply:
Yes, it will. But it will not break although it throws exception.